executors_test.go

```diff
@@ -757,15 +757,11 @@ func Test_TruncateAll(t *testing.T) {
 		ctx, err := tx.Count("users")
 		r.NoError(err)
 		r.Equal(count+1, ctx)
-	})
 
-	transaction(func(tx *Connection) {
-		r := require.New(t)
-
-		err := tx.TruncateAll()
+		err = tx.TruncateAll()
```

Comment:
@stanislas-m I'm trying to think if this is should mess up the testing of any other supported DBs... I know this was important to cockroach, but it seems to have been in since before cockroach support so I'm a bit leery of changing it... that said it seems to pass testing.

Reply:
This is not a part of the fix for cockroach, and just a fix for the test itself.

Please remember, the `transaction()` is an wrapper function to use `Rollback()`. As the result, the test always success regardless of `TruncateAll()` is working or not. (the user added in the first transaction automatically removed by rollback before we call `TruncateAll()`)

I tested it with MySQL, SQLite, and Cockroach before I tried to fix the function for Cockroach since I need to check my fixing result also, but I found its result is always a success, so I fixed it first and checked with same database engines.

Reply:
ah, good point. In my mind it was opening a fresh transaction (and not rolling back unless there was an error). That makes sense now.